spec/src/main/asciidoc/metrics_spec.adoc

```diff
@@ -88,18 +93,40 @@ See section <<app-metrics-api>> for more information regarding the application m
 
 Metrics that are exposed need to be configured in the server. On top of the pure metrics, metadata needs to be provided.
 
-The following three sets of sub-resource (scopes) are exposed.
+The following sets of sub-resource (scopes) are exposed.
 
 * base: metrics that all MicroProfile vendors have to provide
 * vendor: vendor specific metrics (optional)
 * application: application-specific metrics (optional)
+* integration: a place for other MicroProfile specs to expose their data
```

Comment:
I was thinking this is part of application. What end point will this info be exposed under? For FT, I would like to show how many retries per method, when the circuit was open etc. Can all this info be exposed and viewed in Prometheus?

Reply:
yes. It will be up to the respective spec how to populate "their" namespace. Other application specific metrics are using the fully qualified name (by default) of where the annotation is attached.